.gitmodules

```diff
@@ -70,8 +70,8 @@
 	url = https://github.com/px4/rosidl
 [submodule "src/modules/zenoh/zenoh-pico"]
 	path = src/modules/zenoh/zenoh-pico
-	url = https://github.com/px4/zenoh-pico
-	branch = pr-zubf-werror-fix
+	url = https://github.com/eclipse-zenoh/zenoh-pico
```

Comment:
I've pushed dev/1.0.0 to the PX4 fork.
could you change url back to px4/zenoh-pico

Reply:
This has been updated